crates/aptos/CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 
 ### Updated
 - The `--with-faucet` flag has been removed from `aptos node run-local-testnet`, we now run a faucet by default. To disable the faucet use the `--no-faucet` flag.
-- When using `aptos node run-local-testnet` we now expose a transaction stream. Learn more about the transaction stream service here: https://aptos.dev/indexer/txn-stream/. Opt out of this with `--no-txn-stream`.
+- **Breaking change**: When using `aptos node run-local-testnet` we now expose a transaction stream. Learn more about the transaction stream service here: https://aptos.dev/indexer/txn-stream/. Opt out of this with `--no-txn-stream`. This is marked as a breaking change since the CLI now uses a port (50051 by default) that it didn't used to. If you need this port, you can tell the CLI to use a different port with `--txn-stream-port`.
```

Comment:
New port is being used now. Does this break our CLI e2e? (Can't remember if we hard coded the port in e2e)

Reply:
This is an additional new port, all the existing stuff is on the same port.